README.md

```diff
@@ -1,190 +1,99 @@
-	}
-}
-```
-
-##### Joined Statement Workload
-  * `list`, list of other type of workload.
-
-Example:
-```json
-{
-    ...
-	"workload": {
-		"type": "joined",
-        "list": [
-            {
-                "type": "literal"
-                "file": "..."
-                ...
-            }, {
-                "type": "tempalte"
-                "file": "..."
-                ...
-            }, {
-                "type": "tempalte"
-                "file": "..."
-                ...
-            },
-            ...
-        ]
-	}
-}
-```
-
-## The LDBC SNB Workload
-
-### Datagen
-To be added.
-
-### Schema
-```
-CREATE SPACE IF NOT EXISTS ldbc_snb(PARTITION_NUM = 1024)
-
-USE ldbc_snb
-
-CREATE TAG IF NOT EXISTS person(first_name string, last_name string, gender string, birthday, ip string, browser string)
-CREATE TAG IF NOT EXISTS place(name string, type string, url string)
-CREATE TAG IF NOT EXISTS organization(name string, type string, url string)
-CREATE TAG IF NOT EXISTS post(time string, image string, ip string, browser string, language string, content string, length int)
-CREATE TAG IF NOT EXISTS comment(time string, ip string, browser string, content string, length int)
-CREATE TAG IF NOT EXISTS forum(time string, title string, type string)
-CREATE TAG IF NOT EXISTS hashtag(name string, url string)
-CREATE TAG IF NOT EXISTS tagclass(name string, url string)
-
-CREATE EDGE IF NOT EXISTS knows(time string)
-CREATE EDGE IF NOT EXISTS is_part_of()
-CREATE EDGE IF NOT EXISTS person_is_located_in()
-CREATE EDGE IF NOT EXISTS post_is_located_in()
-CREATE EDGE IF NOT EXISTS comment_is_located_in()
-CREATE EDGE IF NOT EXISTS org_is_located_in()
-CREATE EDGE IF NOT EXISTS forum_has_tag(time string)
-CREATE EDGE IF NOT EXISTS comment_has_tag(time string)
-CREATE EDGE IF NOT EXISTS post_has_tag(time string)
-CREATE EDGE IF NOT EXISTS person_has_tag(time string)
-CREATE EDGE IF NOT EXISTS study_at(year string)
-CREATE EDGE IF NOT EXISTS work_at(year string)
-CREATE EDGE IF NOT EXISTS has_post(time string)
-CREATE EDGE IF NOT EXISTS has_comment(time string)
-CREATE EDGE IF NOT EXISTS likes_post(time string)
-CREATE EDGE IF NOT EXISTS likes_comment(time string)
-CREATE EDGE IF NOT EXISTS is_reply_of_post(time string)
-CREATE EDGE IF NOT EXISTS is_reply_of_comment(time string)
-CREATE EDGE IF NOT EXISTS has_member(time string, type string)
-CREATE EDGE IF NOT EXISTS contains_post(time string)
-CREATE EDGE IF NOT EXISTS has_moderator(time string)
-CREATE EDGE IF NOT EXISTS is_tagclass()
-CREATE EDGE IF NOT EXISTS is_sub_of_tagclass()
-```
-
-### Import
-```shell
-$ ./nb --bench-config=ldbc/import/all.json
-```
-
-### Query Benchmark
-```shell
-$ ./nb --bench-config=ldbc/query/3-hop-knows.json
-```
+# 1、Download nebula-bench 
+    
+       https://github.com/vesoft-inc/nebula-bench.git
+ 
+# 2、Download Jmeter
+       cd nebula-bench &&  wget https://mirror.bit.edu.cn/apache//jmeter/binaries/apache-jmeter-5.4.zip  &&  unzip apache-jmeter-5.4.zip 

```

Comment:
why download the jmeter in the dir of nebula-bench. it will contaminate the source directory.

Reply:
This is  a example,user can change dir,I change it.